index.html

```diff
@@ -74,10 +74,29 @@ <h1>Stay Up to Date</h1>
     </div>
 </div>
 
+<div class="grid-container home-sponsors big-bottom-padding">
+    <div class="grid-x grid-padding-x align-center-middle">
+        <div class="large-10 small-10 cell big-bottom-padding">
+            <h1>Keynote Speakers</h1>
+        </div>
+          <div class="large-4 medium-6 small-10 cell big-bottom-padding">
+              <img src="assets/img/matthew_wilkes.jpg" alt="Matthew Wilkes"/>
+          </div>
+          <div class="large-6 small-10 cell ">
+            <h2 class="">Matthew Wilkes</h2>
+            <p class="">Matthew started working with Python in 2005 and has since contributed core code to a number of Python Web frameworks such as  Zope, Plone, Pyramid and Django. For many years he was a member of the Plone security team and served on the Plone Framework team.</p>
```

Comment:
```suggestion
            <p class="">Matthew started working with Python in 2005 and has since contributed core code to a number of Python Web frameworks such as Django, Pyramid and Plone. For many years he was a member of the Plone security team and served on the Plone Framework team.</p>
```

Better like this, I think.